src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -321,6 +321,12 @@ public static FeedbackResponseCommentSearchResultBundle fromResults(
             bundle.commentGiverTable.put(comment.getId().toString(),
                     getFilteredCommentGiverName(bundle, instructorCourseIdList, response, comment, commentGiverName));
             bundle.instructorEmailNameTable.put(comment.giverEmail, commentGiverName);
+            if (!comment.lastEditorEmail.isEmpty()) {
+                InstructorAttributes instructor = instructorsDb.getInstructorForEmail(response.courseId,
```

Comment:
Probably not a good idea to look up the database each time we see an edited comment; perhaps we can check if the email is already in the table to avoid unnecessary lookups?